sbws/lib/v3bwfile.py

```diff
@@ -842,6 +842,17 @@ def to_plt(self, attrs=['bw'], sorted_by=None):
         ys = [[getattr(l, k) for l in self.bw_lines] for k in attrs]
         return x, ys, attrs
 
+    def rm_link(self, output):
```

Comment:
Does sbws need to be able to remove the link?
It seems simpler and more reliable to always write the file, and always have the link.